examples/common/mod.rs

```diff
@@ -10,20 +10,72 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  ********************************************************************************/
+use clap::Parser;
 use up_transport_zenoh::zenoh_config;
 
+#[derive(clap::ValueEnum, Clone, Copy, PartialEq, Eq, Hash, Debug)]
+pub enum Wai {
```

Comment:
`Wai` => `WhatAmI`, right? I can say for certain I would not have understand that if not for having worked with Zenoh a little. Maybe something more descriptive or leave a comment?

Reply:
Agree with you. The reason I used `Wai` is to match the usage in Zenoh examples, but it's unclear as you said.